test/fw/ptl/utils/plugins/ptl_test_runner.py

```diff
@@ -664,6 +704,22 @@ def __are_requirements_matching(param_dic=None, test=None):
         for pk in param_count:
             if param_count[pk] < eff_tc_req[pk]:
                 return False
+        for hostname in param_dic['moms']:
+            pi = system_availability(hostname)
+            sys_avail['min_mom_ram'] = pi.system_ram
+            if eff_tc_req['min_mom_ram'] > sys_avail['min_mom_ram']:
```

Comment:
you can directly use pi.system_ram here no need of sys_avail... And I think condition should be `>=` instead `>`